src/Features/VisualBasic/Portable/CodeFixes/QualifyMemberAccess/VisualBasicQualifyMemberAccessCodeFixProvider.vb

```diff
@@ -0,0 +1,13 @@
+﻿'Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.CodeFixes.Qualify
+
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeFixProviderNames.QualifyMemberAccess), [Shared]>
+    <ExtensionOrder(After:=PredefinedCodeFixProviderNames.RemoveUnnecessaryCast)>
+    Friend Class VisualBasicQualifyMemberAccessCodeFixProvider
```

Comment:
Is this even necessary anymore?  Is it possible to just have the base type and have two ExportCodeFixProviders on it (one for VB and one for C#)?  If that's not possible, then it's fine to have this type.  I'm just ecstatic that it's so empty.